FactoryWeather/App/VIP Settings/SettingsViewController.swift

```diff
@@ -0,0 +1,197 @@
+//
+//  SettingsViewController.swift
+//  FactoryWeather
+//
+//  Created by Matej Korman on 09/01/2019.
+//  Copyright © 2019 Matej Korman. All rights reserved.
+//
+
+import SnapKit
+
+protocol SettingsDisplayLogic: class {
+    func displayLocations(_ locations: [Location])
+    func initializeDisplay(locations: [Location], settings: Settings)
```

Comment:
We should stick to the `displayX` method naming where applicable and sensical - this one is those things. 

`initializeDisplay` -> `displayInitialData`, with the same argument pattern.